src/sql/workbench/contrib/welcome/page/browser/gettingStartedTour.css

```diff
@@ -0,0 +1,371 @@
+	font-weight: 400;
+}
+
+.popup .popup_text_container {
+	padding: 20px 15px 15px 20px;
+}
+
+.popup .popup_text_container p {
+	font-size: 1em;
+}
+
+.popup {
+	position: absolute;
+	width: 400px;
+	left: 45px;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.tour_btn_container {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+}
+
+.tour_btn_container p {
+	font-size: 1.5em;
+	margin: 0;
+	margin-left: 15px;
+}
+
+.tour_btn_container:first-of-type(p) {
+	margin-bottom: 20px;
+}
+
+.popup .btn_exit {
+	background: none;
+	position: absolute;
+	top: 2px;
+	right: 10px;
+	font-size: 1.185em;
+	font-family: inherit;
+	transform: scaleX(1.4);
+}
+
+.popup .btn_exit:hover {
+	cursor: pointer;
+}
+
+.ads_homepage .btn_remove_tour {
+	display: inline-block;
+	width: 12px;
+	height: 12px;
+	-webkit-mask: url(media/close.svg) center center no-repeat;
+	-webkit-mask-size: 12px 12px;
+	mask: url(media/close.svg) center center no-repeat;
+	mask-size: 12px 12px;
+	margin: 0 17px 0 20px;
+}
+
+.popup .btn {
+	box-sizing: border-box;
+	width: 77px;
+	height: 34px;
+	padding-top: 1px;
+	font-size: 1.08em;
+	line-height: 29px;
+	border-radius: 2px;
+	display: flex;
+	align-content: center;
+	justify-content: center;
+	text-align: center;
+	text-decoration: none;
+	margin: 0;
+	align-self: flex-end;
+	cursor: pointer;
+}
+
+.popup .flex.column {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	height: 100%;
+}
+
+.popup h1, .popup p {
+	font-family: Segoe UI, Frutiger, Frutiger Linotype, Dejavu Sans, Helvetica Neue, Arial, sans-serif;
+	font-weight: 400;
+}
+
+.popup h1 {
+	margin: 0;
+	font-size: 20px;
+}
+
+.popup.arrow_left:after {
+	content: " ";
+	position: absolute;
+	left: -10px;
+	top: 8px;
+	border-top: 10px solid transparent;
+	border-left: none;
+	border-bottom: 10px solid transparent;
+}
+
+.popup.arrow_top:after {
+	content: " ";
+	position: absolute;
+	left: 0;
+	right: 0;
+	border-top: 10px solid transparent;
+	border-left: none;
+	border-bottom: 10px solid transparent;
+	transform: rotate(90deg);
+	bottom: 509px;
+}
+
+.popup.arrow_bottom_left:after {
+	content: " ";
+	position: absolute;
+	left: -10px;
+	border-top: 10px solid transparent;
+	border-left: none;
+	border-bottom: 10px solid transparent;
+	bottom: 3px;
+}
+
+.popup.arrow_center_left:after {
+	content: " ";
+	position: absolute;
+	left: -10px;
+	border-top: 10px solid transparent;
+	border-left: none;
+	border-bottom: 10px solid transparent;
+	bottom: 145px;
+}
+
+.popup {
+	max-height: 361px;
+	display: none;
+	z-index: 1;
+}
+
+.popup.show {
+	display: block;
+}
+
+.tour_overlay {
+	width: 100%;
+	height: 100%;
+	position: absolute;
+	top: 30px;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	opacity: .6;
+}
+
+.tour_btn_container .docs_link {
+	align-self: center;
+	justify-self: flex-start;
+	margin-right: auto;
+}
+
+.btn {
```

Comment:
Stuff like this should be scoped down - btn is way too generic of a class and I assume this isn't meant to apply outside of the tour panel

Reply:
This applies to a lot of the rules in this file. Anything that isn't specifically scoped down to be under ads_tour should really either go somewhere else or have some other specific class that's applied during the tour.

Reply:
fixed